back/spec/services/projects_finder_admin_service_spec.rb

```diff
@@ -0,0 +1,209 @@
+require 'rails_helper'
+
+describe ProjectsFinderAdminService do
+  def create_project(
+    start_at,
+    end_at,
```

Comment:
Minor nitpick, but use of this method might be clearer if these two were also named variables:
```suggestion
    start_at: nil,
    end_at: nil,
```